backend/controllers/houseblockList.js

```diff
@@ -22,20 +31,29 @@ const getHouseblockListFactory = (elite2Api) => {
       let currentRow = rows[i];
       if (event.eventType === 'PRISON_ACT') {
         if (!currentRow.activity) {
-          currentRow.activity = [event];
+          // 1st activity found for this offender
+          currentRow.activity = event;
         } else {
-          currentRow.activity.push(event);
-          currentRow.activity.sort(function (a, b) {
-            return safeTimeCompare(a, b);
-          });
+          // Multiple activities - make paid the main one
+          if (!currentRow.activity.payRate && event.payRate) {
+            addToOthers(currentRow, currentRow.activity);
+            currentRow.activity = event;
+          } else if ((currentRow.activity.payRate && event.payRate) ||
```

Comment:
Could this just be an else? It seems to be true if null or not null.

Reply:
Actually, my comment isn't true. You checking if both are true or both are false. I wonder if there's a cleaner way to describe the meaning of the different combinations.

Reply:
possibly. Ill have a think and if so  commit it with the next ticket